lib/AST/Decl.cpp

```diff
@@ -5142,6 +5146,10 @@ bool NominalTypeDecl::isResilient() const {
   return getModuleContext()->isResilient();
 }
 
+bool NominalTypeDecl::isStrictlyResilient() const {
```

Comment:
Nitpick: `strict` is vague when used as an API name. A better alternative could be something like `isNonResilientAccessDisallowed()`